Makefile

```diff
@@ -1,8 +1,12 @@
-.PHONY: build-server build-agent test clean run-server run-agent lint fmt deps
+.PHONY: build-server build-agent test clean run-server run-agent lint fmt deps atlas-migration goose-up goose-down
 
 # Define Go command, which can be overridden
 GO ?= go
 
+include server.env
```

Comment:
[SUGGESTION] You can use `Makefile.local` to overwrite env variables.

Reply:
Fixed